guides/source/configuring.md

```diff
@@ -500,7 +500,7 @@ The default value depends on the `config.load_defaults` target version:
 
 #### `config.public_file_server.enabled`
 
-Configures Rails to serve static files from the public directory. This option defaults to `true`, but in the production environment it is set to `false` because the server software (e.g. NGINX or Apache) used to run the application should serve static files instead. If you are running or testing your app in production using WEBrick (it is not recommended to use WEBrick in production), set the option to `true`. Otherwise, you won't be able to use page caching and request for files that exist under the public directory.
+Configures whether Rails should serve static files from the public directory. This option defaults to true for all environments. If the server software (e.g. NGINX or Apache) used to run the application in the production environment should serve static files instead, set this value to false. If you are running or testing your app in production using WEBrick (it is not recommended to use WEBrick in production), set the option to true. Otherwise, you won't be able to use page caching and request for files that exist under the public directory.
```

Comment:
Good catch!  I think we could also remove the part about setting to `true` for WEBrick:

```suggestion
Configures whether Rails should serve static files from the public directory. 
Defaults to `true`.

If the server software (e.g. NGINX or Apache) should serve static files instead, 
set this value to `false`.
```

<!-- Ref: https://github.com/rails/rails/commit/6d965355244055e7bef995f1a263cb1cd2993e45 -->